README.md

```diff
@@ -12,12 +12,14 @@ This service will help service provider to set up DFT/SDE with EDC and EDC as se
 ### Software Version
 
 ```shell
-Helm release version: 1.1.4
-Image version: 1.1.4
+Application version: 1.1.4
+Helm release version: 1.1.5
 ```
 
 
-##### For installation guide: see [InstallationGuide.md](InstallationGuide.md)
+##### For installation guide:
+
+https://github.com/eclipse-tractusx/autosetup-backend/blob/main/INSTALL.md
```

Comment:
this needs to be linked relative otherwise you always point to the newest installation guide: used use INSTALL.md or [INSTALL.md](INSTALL.md)

Reply:
added relative path